### ISSUE TITLE
[FEATURE] Add a Like or Upvote button for each project

### ISSUE THREAD

**Issue by rupali-codes** [rohitdasu/projectmate#305]
### Description

It would be nice having a Like or upvote button for project, so user's of the site would be able like the project, also it will put a soft impact on visitors of the project.

**Comment by rohitdasu** [comment]
@rupali-codes can we split this issue with [FRONTEND] and [BACKEND]?

**Comment by rupali-codes** [comment]
@rohitdasu you mean tags, right?

**Comment by rohitdasu** [comment]
yes, like rename this issue having a frontend tag example `[feature] add a upvote or like button - Frontend` and create another issue for backend. let me know if you have any doubts.

**Comment by rupali-codes** [comment]
Alright @rohitdasu, Thanks.

**Comment by Olaleye-Blessing** [comment]
I will like to take up this issue.

**Comment by rupali-codes** [comment]
@Olaleye-Blessing thanks for taking interest, you can start working on it, and create pull request. Let us know for any kind of issues.

**Comment by zvolcsey** [comment]
I also created a frontend version, but I can push my work without frontend if there is a better version.

**Comment by zvolcsey** [comment]
With a frontend version I was able to visually test my backend code in development mode.

**Comment by rohitdasu** [comment]
Did you create a seperate branch for fronted implementation? @zvolcsey
